recipes/libressl/2.9.2/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibreSSLConan(ConanFile):
+    name = "libressl"
+    version = "2.9.2"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.libressl.org/"
+    license = "OpenSSL - OpenBSD"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = "shared=False", "fPIC=True"
+    topics = ("SSL", "TLS", "openssl")
+    description = "LibreSSL is a version of the TLS/crypto stack forked from OpenSSL in 2014, with goals of modernizing the codebase, improving security, and applying best practice development processes."
+    generators = "cmake"
+
+    @property
+    def libressl_src(self):
+        return "%s-%s" % (self.name, self.version)
+
+    def config_options(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+
+    def build(self):
+        # needed to properly overwrite MSVC runtime
+        tools.replace_in_file(os.path.join(self.libressl_src, "CMakeLists.txt"), "project (LibreSSL C ASM)", """project (LibreSSL C ASM)
+include(${CMAKE_CURRENT_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)""")
+
+        # libressl adds a suffix to Windows dll names, which is really annoying for packaging
+        tools.replace_in_file(os.path.join(self.libressl_src, "crypto", "CMakeLists.txt"), "set(CRYPTO_POSTFIX -${CRYPTO_MAJOR_VERSION})", "set(CRYPTO_POSTFIX)")
+        tools.replace_in_file(os.path.join(self.libressl_src, "tls", "CMakeLists.txt"), "set(TLS_POSTFIX -${TLS_MAJOR_VERSION})", "set(TLS_POSTFIX)")
+        tools.replace_in_file(os.path.join(self.libressl_src, "ssl", "CMakeLists.txt"), "set(SSL_POSTFIX -${SSL_MAJOR_VERSION})", "set(SSL_POSTFIX)")
+        cmake = CMake(self)
+        cmake.definitions["BUILD_SHARED"] = self.options.shared
+        cmake.definitions["USE_SHARED"] = self.options.shared
+        # LibreSSL's CMake install rules are a bit weird
+        # explicitly remove CMAKE_INSTALL_PREFIX to avoid installing certs
```

Comment:
Or, you can install and remove what is restricted.

Reply:
I shall change the comment, there are no certs installed, but it will embed an absolute path in the build libraries if the prefix is not removed